spec/request/api/v1/targets/destroy_spec.rb

```diff
@@ -0,0 +1,53 @@
+describe 'DELETE api/v1/targets/:id', type: :request do
+  let(:user)               { create(:user) }
+  let!(:target)            { create(:target, user: user) }
+  let(:other_users_target) { create(:target) }
+  let(:unexisting_target)  { 0 }
+
+  context 'for an existing target' do
+    context 'when deleting an own target' do
+      subject do
+        delete api_v1_target_path(target.id), headers: auth_headers, as: :json
+        response
+      end
+
+      it 'returns a successful response' do
+        expect(subject).to have_http_status(:success)
+      end
+
+      it 'deletes the target' do
+        expect { subject }.to change(user.targets, :count).by(-1)
+      end
+    end
+
+    context 'when deleting other user\'s target' do
+      subject do
+        delete api_v1_target_path(other_users_target.id), headers: auth_headers, as: :json
+        response
+      end
+
+      it 'does not return a successful response' do
+        expect(subject).not_to have_http_status(:success)
```

Comment:
Maybe we should expect some http status instead of expecting any status different from `success`, I think it should be `unauthorized`